docs/source/intro.rst

```diff
@@ -31,7 +31,7 @@ compute the analytic derivatives of functions. Suppose we want to compute this d
     Tensor(9.0)
 
 
-Invoking :meth:`~mygrad.Tensor.backward` on ``f`` instructs ``mygrad`` to trace through the computational graph that produced ``f`` and compute the
+Invoking :math:`~mygrad.Tensor.backward` on ``f`` instructs ``mygrad`` to trace through the computational graph that produced ``f`` and compute the
```

Comment:
```suggestion
Invoking :meth:`~mygrad.Tensor.backward` on ``f`` instructs ``mygrad`` to trace through the computational graph that produced ``f`` and compute the
```
`meth` is correct here. It signifies to Sphinx that this is a class-method

Reply:
man I even grepped for :meth: to make sure